src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -155,43 +186,34 @@ public long amount() throws IOException {
 
     @Override
     public String prefix() throws IOException {
-        try {
-            final String prefix = new ItemAt<>(
-                new SplitText(
-                    new TextOf(
-                        this.transaction
-                    ),
-                    new TextOf(";")
-                ).iterator(),
-                //@checkstyle MagicNumberCheck (1 line)
-                3
-            ).value().asString();
-            //@checkstyle MagicNumberCheck (1 line)
-            if (prefix.length() < 8 || prefix.length() > 32) {
-                throw new IllegalArgumentException("Invalid prefix size");
-            }
-            if (!RtTransaction.TRANSACTION.matcher(prefix).matches()) {
-                throw new IllegalArgumentException("Invalid base64 prefix");
-            }
-            return prefix;
-            //@checkstyle IllegalCatchCheck (1 line)
-        } catch (final Exception exception) {
-            throw new IOException(
-                "Invalid prefix string",
-                exception
-            );
+        final String prefix = new UncheckedText(
+            new IoCheckedScalar<String>(
+                () -> new ItemAt<Text>(
+                    //@checkstyle MagicNumberCheck (1 line)
+                    3, new SplitText(this.transaction.value(), ";")
+                ).value().asString()
+            ).value()
+        ).asString();
+        //@checkstyle MagicNumberCheck (1 line)
+        if (prefix.length() < 8 || prefix.length() > 32) {
+            throw new IOException("Invalid prefix size");
+        }
+        if (!RtTransaction.TRANSACTION.matcher(prefix).matches()) {
+            throw new IOException("Invalid base64 prefix");
         }
+        return prefix;
     }
 
     @Override
     public String bnf() throws IOException {
-        final String bnf =
-            new IoCheckedScalar<>(
-                () -> new ItemAt<>(
-                    // @checkstyle MagicNumberCheck (1 line)
-                    4, new SplitText(this.transaction, ";")
+        final String bnf = new UncheckedText(
+            new IoCheckedScalar<String>(
+                () -> new ItemAt<Text>(
```

Comment:
@paulodamaso no need to use `UncheckedText` here, we can avoid it and the re-wrapping of `String` into `IoCheckedScalar` by changing `transaction` to `IoCheckedScalar`.